def raise_exception_msg():
    raise NameError()